Abracadabra/AbracadabraAPI/AbracadabraAPI/Controllers/DashboardController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
```

Comment:
Why's the dashboad controller made in this branch?

Reply:
It was still in the begining phases of getting created and I managed to push it, it isnt really needed. But I don't think it will cause problems.